mscore/scorecmp/scorecmp.cpp

```diff
@@ -240,7 +240,7 @@ void ScoreComparisonTool::updateDiff()
 
 Score* ScoreComparisonTool::openScoreVersion(const ScoreVersion& ver)
       {
-      Score* s;
+      Score* s = 0;
```

Comment:
`nullptr` is more effective.

Reply:
well, its the same thing to C/C++ and less to type. And on occasions where I did use nullptr it didn't pass review ;-)

Reply:
but as you want it: here we go